ext/POSIX/t/posix.t

```diff
@@ -292,7 +292,9 @@ SKIP: {
 		    `cl --version 2>&1` =~ /^.*Version\s+([\d.]+)/ and $1 >= 14)
 		or (Win32::GetOSVersion())[1] >= 6));
 
-    try_strftime("Thu Feb 29 00:00:60 1996 060", 60,0,-24, 30,1,96);
+    # This is an actual time a leap second occurred.
+    # https://en.wikipedia.org/wiki/Leap_second */
+    try_strftime("Mon Jun 30 23:59:60 1997 181", 60,59,23, 30,5,97);
```

Comment:
strftime() works in local time, but leap seconds are added at midnight UTC, not localtime.

So this change only matters when the current time zone is UTC (Britain in winter, some parts of Africa I assume)

Reply:
So my analysis was flawed.  

Now updated to detect `mktime`  not handling leap seconds like `mini_mktime` does and to override with the latter